www/app/App.test.js

```diff
@@ -1,11 +1,11 @@
 import React, { useEffect } from 'react';
 import { render } from '@testing-library/react';
 import { App } from './App';
-import { CookieBanner } from '../shared/components';
+import CookieBanner from '../shared/modules/react-cookie-banner';
 import { initGTM, destroyGTM } from '../shared/utils/google-tag-manager';
-import { AppTreeWrapper } from '../shared/test-utils/components';
+import { AppTreeWrapper } from '../shared/test-utils/modules';
```

Comment:
I think we should follow the same approach we have in `shared/modules` and remove the index. What do you guys think?

Reply:
`react-app-tree`

Reply:
I think it would be better to keep consistency.